### ISSUE TITLE
Cost & Calculator

### ISSUE THREAD

**Issue by Colin-TUE** [Brutusn/ScoutingVeghel#21]
Add the cost for renting the blokhut in the description and add a small calculator to compute the costs as indication. 
- [x] Added cost description
- [ ] Added cost calculator

**Comment by Brutusn** [comment]
De kosten zelf zijn belangrijker dan de calculator.

**Comment by Colin-TUE** [comment]
I agree, so lets just put the costs there and leave the calculator for now?

**Comment by Brutusn** [comment]
Jup, but leave the issue open.

**Comment by Colin-TUE** [comment]
Calculator zal niet in deze release en dus kan hij uit de Eerste Release
